css/index.css

```diff
@@ -1,3 +1,285 @@
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+/* code starts here */
+.container {
+	width: 1000px;
+    width: 100%;
+	max-width: 1000px;
+	margin: auto;
+	display: flex;
+	flex-direction: column;
+}
 
+header {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    padding: 3% 0;
+
+}
+
+header nav{
+    display: flex;
+    justify-content: space-around;
+    width: 70%;
+    justify-content: space-around;
+    align-items: center;
+
+}
+
+header nav a {
+    text-decoration: none;
+    margin: 20px;
+    color: grey;
+
+}
+header nav a:hover {
+	color: black;
+}
+
+header .logo {
+	margin: 0 50px;
+    height: 100%;
+    width: 22%;
+    margin-left: 5%;
+}
+
+.top-content-left h1 {
+	font-size: 90px;
+	
+}
+.top-content {
+	display: flex;
+    justify-content: space-between;
+    padding: 0 40px;
+}
+
+.top-content-left {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
+	text-align: center;
+	margin: 20px;
+	
+}
+.top-content-right {
+	margin: 20px
+	
+}
+
+.top-content-right img {
+	margin-top: 50px;
+    margin-top: 50px;
+    width: 100%
+}
+
+.top-content-left button {
+	padding: 10px 0px;
+	width: 40%;
+	background: white;
+}
+.top-content-left button:hover {
+	color: white;
+	background: grey;
+	cursor: pointer;
+}
+.middle-content {
+	display: flex;
+	flex-direction: column;
+}
+.middle-img {
+    margin: 0 30px;
+}
+.middle-content-top {
+	display: flex;
+	justify-content: space-between;
+	border-top: 2px solid black;
+	margin: 30px;
+	padding-top: 30px;
+}
+.features {
+	width: 50%;
+	
+}
+.about {
+	width: 50%;
+}
```

Comment:
You can also target this together using 
`.middle-content-top > div {}`

Reply:
or combining the selectors since they have the same style rules
`.features .about { width: 50%}`